apps/astarte_pairing/lib/astarte_pairing/fdo/service_info.ex

```diff
@@ -67,4 +72,81 @@ defmodule Astarte.Pairing.FDO.ServiceInfo do
   def generate_msg_67(payload) do
     CBOR.encode(payload) |> COSE.tag_as_byte()
   end
+
+  # first device message, sending devmod
+  def handle_message_68(
+        realm_name,
+        session_max_service_info,
+        session_remaining,
+        %DeviceServiceInfo{is_more_service_info: false, service_info: service_info} =
+          device_service_info
+      ) do
+    devmod_data = %{
+      active: Map.fetch!(service_info, "devmod:active"),
+      sn: Map.fetch!(service_info, "devmod:sn"),
+      n_modules: Map.fetch!(service_info, "devmod:nummodules"),
+      modules: Map.fetch!(service_info, "devmod:modules")
+    }
+
+    with {:ok, credentials_secret} <- Engine.register_device(realm_name, devmod_data.sn),
+         {:ok, device_id} <- Device.decode_device_id(devmod_data.sn, allow_extended_id: true),
+         {:ok, session} <- Session.fetch(realm_name, session_key) do
+      # max_service_info
+
+      owner_service_info = %OwnerServiceInfo{
+        is_more_service_info: false,
+        is_done: true,
+        service_info: %{
+          "astarte:active": true,
+          "astarte:realm": realm_name,
+          "astarte:secret": credentials_secret,
+          "astarte:baseurl": "api.#{Config.base_url_domain!()}",
```

Comment:
`Config.base_url_domain!()` is [this](https://github.com/astarte-platform/astarte/blob/18319fd8e6f4ff5171955b2e7669d92d5f7c42b5/apps/astarte_pairing/lib/astarte_pairing/config.ex#L48-L50): 
> The astarte base domain, used for FDO authentication mechanism

I don't think that `api` should be prefixed here